API/NoDocs/Aspire.Azure.Messaging.EventHubs/AzureMessagingEventHubsConsumerSettings.cs

```diff
@@ -0,0 +1,8 @@
+// Assembly 'Aspire.Azure.Messaging.EventHubs'
+
+namespace Aspire.Azure.Messaging.EventHubs;
+
+public sealed class AzureMessagingEventHubsConsumerSettings : AzureMessagingEventHubsConsumerBaseSettings
```

Comment:
Did we talk with any of the EH folks about whether to use `EventHub` or `EventHubs`?  We use plural `EventHubs` in the package/namespace but it's singular in `EventHubConsumerClient`.